invenio_app_ils/acquisition/config.py

```diff
@@ -142,38 +89,25 @@
             order=5,
         ),
     ),
-    acq_vendors=dict(  # VendorSearch.Meta.index
-        name=dict(fields=["name.keyword"], title="Name", order=1),
-        created=dict(
-            fields=["_created"],
-            title="Recently added",
-            order=2,
-        ),
-        bestmatch=dict(
-            fields=["-_score"],
-            title="Best match",
-            order=3,
-        ),
-    ),
 )
 
-FACET_VENDOR_LIMIT = 5
+FACET_PROVIDER_LIMIT = 5
 
 RECORDS_REST_FACETS = dict(
     acq_orders=dict(  # OrderSearch.Meta.index
         aggs=dict(
             status=dict(terms=dict(field="status")),
-            vendor=dict(
+            provider=dict(
                 terms=dict(
-                    field="vendor.name.keyword", size=FACET_VENDOR_LIMIT
+                    field="provider.name.keyword", size=FACET_PROVIDER_LIMIT
```

Comment:
I didn't remember this. @kprzerwa maybe we can add this to limit the number of languages in the frontsite facet for the time being?

Reply:
we will push the languages down as a facet, but before we have "show more" for facets I wouldn't limit the languages on the frontsite, I think it will be confusing for the users